server/sonar-server/src/test/java/org/sonar/server/qualitygate/QgateProjectFinderTest.java

```diff
@@ -89,12 +91,11 @@ public void return_empty_association() {
 
   @Test
   public void return_all_projects() {
-    OrganizationDto org = dbTester.organizations().insert();
```

Comment:
it's better to create the organization in each test, in order to be sure it's used